lib/python/README.md

```diff
@@ -0,0 +1,51 @@
+Check out the DIDKit documentation [here](https://spruceid.dev/docs/didkit/).
+
+# DIDKit
+
+DIDKit provides Verifiable Credential and Decentralized Identifier
+functionality across different platforms. It was written primarily in Rust due
+to Rust's expressive type system, memory safety, simple dependency web, and
+suitability across different platforms including embedded systems. DIDKit
+embeds the [`ssi`](https://github.com/spruceid/ssi) library, which contains the
+core functionality.
+
+![DIDKit core components](https://spruceid.dev/assets/images/didkit-core-components-7abba2778ffe8dde24997f305e706bd8.png)
+
+## Building
+
+Make sure you have the latest versions of pip and PyPA’s build installed:
+```bash
+sudo apt install -y python3-pip python3-virtualenv
+python3 -m pip install --upgrade pip build
+```
+
+Build DIDKit:
+```bash
+cargo build --release
+```
+
+Build the package
+```bash
+python3 -m build
+```
+
+Install the package, you can find the `$VERSION` [here](./setup.cfg)
```

Comment:
Maybe add a command to load the version into the variable.
```
export VERSION=$(cat setup.cfg | grep version | cut -d' ' -f3)
```

Reply:
Appreciated, added to the Makefile also.